public/manifest.json

```diff
@@ -1,21 +1,11 @@
 {
-  "short_name": "React App",
```

Comment:
is this related to this PR

Reply:
I kept getting 404's for the logo's in the manifest file so I removed the references and figured I would update the names while at it.

Reply:
👍 it would be good to have separate PR for this to keep git history clean w/ single purpose commit next time.